netmiko/huawei/huawei.py

```diff
@@ -159,7 +160,9 @@ def special_login_handler(self, delay_factor: float = 1.0) -> None:
             r"security\srisks\sin\sthe\sconfiguration\sfile.*\?\s*\[[Yy]\/[nN]\]", data
         ):
             self.send_command("Y", expect_string=r"continue\?\s*\[[Yy]\/[nN]\]")
-            self.send_command("Y", expect_string=r"saved\ssuccessfully", read_timeout=60)
+            self.send_command(
+                "Y", expect_string=r"saved\ssuccessfully", read_timeout=60
+            )
```

Comment:
Huawei, should not be included in commit?
Also time is imported but unused after